tests/test_octodns_provider_ns1.py

```diff
@@ -1198,6 +1199,138 @@ def test_extra_changes(self, monitors_for_mock):
         self.assertFalse(extra)
         monitors_for_mock.assert_not_called()
 
+    def test_has_dynamic(self):
+        provider = Ns1Provider('test', 'api-key')
+
+        desired = Zone('unit.tests.', [])
+
+        simple = Record.new(desired, 'sim', {
```

Comment:
Should we move this and the dynamic record below to a function - getSimpleTestRecord() and getDynamicTestRecord() and use as needed? It exists in the current code and duplicated twice in the new tests in this change.